### PR TITLE
Topic watsh switch to graalvm

### PR DESCRIPTION
**Pull request by watsh-rajneesh**
Changes to use graalVM Oracle GraalVM Enterprise Edition 20.1.0.1 Linux x86 for Java 11 as Java runtime in elasticsearch docker image.

Acceptance test :  https://build.verrazzano.io/job/verrazzano-remote/job/watsh_bfstest/7/console using       --set monitoringOperator.esImage="phx.ocir.io/stevengreenberginc/bfs/elasticsearch:7.6.1-2-watsh" passed successfully.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -13,11 +13,18 @@ RUN gradle clean \
 # unzip in the build stage; tar/zip is not available by default in OL7 slim
 RUN mkdir /usr/share/elasticsearch && tar -xzf distribution/archives/oss-linux-tar/build/distributions/elasticsearch-oss-${ELASTIC_VERSION}-linux-x86_64.tar.gz --strip-components=1 --directory=/usr/share/elasticsearch
 
+# Replace the JDK
```
Comment (mgianatagh) [inline Dockerfile:16]: The from image is gradle:6.2.2-jdk13, is that an Oracle built image?

Comment (watsh-rajneesh) [reply Dockerfile:16]: Hi Mike, Mark, changing the build stage image from gradle6.2.2-jdk13 is not straight forward but I will continue on it today.

Comment (watsh-rajneesh) [reply Dockerfile:66]: I have changed the from image to be built with OL7-slim+gradle-6.2.2+GraalVM11.0.7
Please review again.

File: Dockerfile [hunk 2]
```diff
@@ -1,25 +1,79 @@
...
+ENV LANG=en_US.UTF-8 LANGUAGE=en_US:en LC_ALL=en_US.UTF-8
+ENV ES_BUNDLED_JDK false
+ENV ES_JAVA_OPTS "-Des.bundled_jdk=false"
+
+# Environment variables for the builder image.
+# Required to validate that you are using the correct file
+
+ENV	JAVA_HOME=/usr/java
+
+COPY --from=graalvm-docker-staging-local.dockerhub-den.oraclecorp.com/graalvm/enterprise-11:jdk-20.1.0.1-b1 /usr/lib64/graalvm/graalvm20-ee-java11 /usr/java/
```
Comment (markxnelson) [inline Dockerfile:26]: remove internal system name

Comment (watsh-rajneesh) [reply Dockerfile]: Fixed.

**Comment by watsh-rajneesh** [comment]
> fix base image, as per mike's comment please

Done.

**Comment by markxnelson** [comment]
acc test confirmed

**Pull request merged**